src/networks.json

```diff
@@ -554,15 +574,5 @@
     "multicall": "0x6F522a3982e8F9A50A2EDc9E46ed1A3aE2B3FD3a",
     "rpc": ["https://evm-cronos.crypto.org"],
     "explorer": "https://cronos.crypto.org/explorer"
-  },
```

Comment:
Could you remove changes here?

Reply:
ok

Reply:
@bonustrack I changed the code, can you check?